rpm/authselect.spec.in

```diff
@@ -70,7 +70,14 @@ Requires: gawk
 Requires: grep
 Requires: sed
 Requires: systemd
-Requires: pam >= 1.3.1-23
+%if %{enforce_authselect}
+# authselect now owns nsswitch.conf (glibc) and pam files
+Conflicts: pam < 1.5.2-8
```

Comment:
This requirement was added in 41b7f732eb30e790746e0c2966bad920b8cf3398 since we started to use a new module pam_usertype therefore we used this requires to ensure that a pam gets updated with authselect as well. It is no longer necessary (and it probably should have been solved by conflicts as well).

It does not make sense to use authselect without pam, obviously (it is also not possible to uninstall pam), but authselect itself imho does not need to depend on pam as it does not use any pam api and does not call pam in any way.

Neither pam, systemd, nss-mdns or glibc is used by authselect (well, glibc is of course used but it is not needed in spec file). These Conflicts are here to ensure that when updating to new authselect, other packages are updated as well. But at the same time, you can install authselect without having systemd/nss-mdns installed.